### PR TITLE
Profiler example fix

### LINKED ISSUE #1044: Profiler example documentation does not work
If someone follows the current documentation (s)he would not be able to run through the example successfully

### PR DESCRIPTION
**Pull request by repenno**
Fixes #1044

If someone follows the current documentation they would not be able to run through the example. This fixes it.

### CODE REVIEW

File: docs/book/profiler.md [hunk 1]
```diff
@@ -28,19 +28,23 @@ while `line` has the **lowest**.
 
 #### Example Policy
 The different profiling examples shown later on this page use the below
-sample policy.
+sample policy. In order to run this example, create a directory called `foo` and inside a file called baz.rego. Copy and paste the entire contents of the example below to the file and save it.
```
[the remark this answers is not in the document]
Comment (repenno) [reply docs/book/profiler.md]: The file name does not matter, therefore I chose a name that is different from the package name on purpose so people know they do not need to be the same.

[the remark this answers is not in the document]
Comment (srenatus) [reply docs/book/profiler.md]: Sorry, I was referring to the profiling output below, like [the table here](https://github.com/open-policy-agent/opa/blob/b19e9e6a956277a72feaaa8ffcfa6ab8d962fe1c/docs/book/profiler.md#example-display-all-profile-results-with-default-ordering-criteria) which has a rego filename in it.

[the remark this answers is not in the document]
Comment (repenno) [reply docs/book/profiler.md]: oh right, I will fix it

**Comment by tsandall** [comment]
This is kind of funny. It looks like when we merged the profiler changes we didn't fix the docs properly. The profiler docs are contained in docs/book/how-do-i-test-policies.md. This file should just be deleted. @repenno sorry for the confusion. Can you please amend your PR to just remove this file entirely? Thanks!

**Comment by repenno** [comment]
The profile section under how-do-i-test-policies.md has the same problems as this one, it would need to be fixed as well. How about adding a link to this document in how-do-i-test-policies.md since it is self-contained and we can update easier?

**Comment by tsandall** [comment]
@repenno I don't think the actual version has a problem. If you want to modify the line before the example go ahead, but the `opa eval` commands (and their output) appear to  be correct.

Current:
```
The different profiling examples shown later on this page use the below sample policy.
```

Desired (fixed wording & added filename):
```
The examples below use the following policy (**rbac.rego**):
```

**Pull request merged**